lib/fileProcessor.js

```diff
@@ -0,0 +1,59 @@
+const moment = require('moment');
+class FileProcessor {
+  constructor(store, files) {
+    this.store = store;
```

Comment:
suggest removing for clarity and simplicity.  In this case each processor needs its own set of data.  The FileProcessor needs the file list but not a store.  The GitHubProcessor needs a store, ...